cilium/ebpf.io-summit
# feat: add Datoms logo to Members section

**Describe what changes this pull request brings**
This pull request adds the logo Datoms to Silver Members on the Foundation page.


**Describe any additional changes**


**Steps to test**
1. Open [the page](https://deploy-preview-24--ebpf-summit-2021.netlify.app)
2. Confirm that everything looks and works as expected


**Screenshots**
![image](https://user-images.githubusercontent.com/48465000/141982017-e3dcf3eb-2522-47a3-bc3f-93a5ccb8d58b.png)



**Checklist**



**TODO**


**References**
